.github/workflows/wheel.yml

```
@@ -18,7 +18,9 @@ on:
 jobs:
   wheel-build:
     name: Python wheel build
-    if: github.event.review.state == 'approved'
+    # Only the third condition, this job is not executed because the condition is always false
+    # if this job is triggered by 'push' or workflow_dispatch.
```

五月雨式になって申し訳ないのですが、こちらのコメントも変更お願いします。
```suggestion
    # For jobs triggered by pull_request_review, build task should run only if is in `approved` state.
```